README.md

```diff
@@ -1,10 +1,21 @@
 # node-problem-detector
 node-problem-detector aims to make various node problems visible to the upstream
-layers in cluster management stack. It is a [DaemonSet](http://kubernetes.io/docs/admin/daemons/)
-detecting node problems and reporting them to apiserver. Now it is running as
+layers in cluster management stack.
+
+`node-problem-detector` can either run as a [DaemonSet](http://kubernetes.io/docs/admin/daemons/) or run standalone on bare metals.
+
+`node-problem-detector` detects node problems and reporting them to apiserver.
+
+Now it is running as
 a [Kubernetes Addon](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons)
 enabled by default in the GCE cluster.
 
+# Command Line Arguments
+* `apiserver`  
+`apiserver` command line argument can customize how to generate a Kubernetes ApiServer client according to `inClusterConfig` URI parameter.
```

Comment:
We should move this into `Usage`. Probably add a "Override Apiserver Client Configuration" section